src/main.rs

```diff
@@ -229,3 +289,16 @@ fn ram_x() {
 fn ram_y() {
     dbg!(ram_y as fn() as *const ());
 }
+
+#[link_section = ".data"]
+#[interrupt]
+fn SIO_IRQ_PROC0() {
```

Comment:
To document https://github.com/DusterTheFirst/pico-dvi-rs/pull/12#discussion_r1305013352

```suggestion
/// Called by the system only when core 1 is overloaded and can't handle all the rendering work, and requests core 0 to render one scan line worth of content.
#[link_section = ".data"]
#[interrupt]
fn SIO_IRQ_PROC0() {
```